src/components/MessageIcons.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import EditIcon from '../assets/images/icons/edit-icon.png';
+import PinIcon from '../assets/images/icons/pin.png';
+import TrashIcon from '../assets/images/icons/trash-icon.png';
+import theme from '../styles/theme.style.js';
+import { Container, Button, View, TouchableOpacity, Text, Image, Alert, StyleSheet} from 'react-native';
+
+const createTwoButtonAlert = () =>
+Alert.alert(
+  "Are you sure?",
+  "Do you really want to delete the message?",
+  [
+    {
+      text: "Cancel",
+      onPress: () => console.log("Cancel Pressed"),
+      style: "cancel"
+    },
+    { text: "Yes", onPress: () => console.log("OK Pressed") }
+  ]
+);
+class MessageIcons extends Component {
+
+
+  render() {
+    return (
+      <MessageIconsContainer>
+              <ClickEdit onPress={this.onButtonPress}>
```

Comment:
fix indentation of ClickEdit, ClickTrash and ClickPin

Reply:
Done!